src/components/organisms/RecentTradesTable/RecentTradesTable.tsx

```diff
@@ -0,0 +1,239 @@
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableColumn,
+    TableHeader,
+    TableRow,
+} from '@nextui-org/table';
+import { OrderSide } from '@secured-finance/sf-client';
+import { toBytes32 } from '@secured-finance/sf-graph-client';
+import queries from '@secured-finance/sf-graph-client/dist/graphclients';
+import clsx from 'clsx';
+import dayjs from 'dayjs';
+import { useEffect, useMemo, useState } from 'react';
+import ArrowUpSquare from 'src/assets/icons/arrow-up-square.svg';
+import ShowFirstIcon from 'src/assets/icons/orderbook-first.svg';
+import ShowAllIcon from 'src/assets/icons/orderbook-full.svg';
+import ShowLastIcon from 'src/assets/icons/orderbook-last.svg';
+import { Spinner } from 'src/components/atoms';
+import {
+    useBlockExplorerUrl,
+    useGraphClientHook,
+    useIsSubgraphSupported,
+} from 'src/hooks';
+import useSF from 'src/hooks/useSecuredFinance';
+import { currencyMap, formatLoanValue, ordinaryFormat } from 'src/utils';
+import { LoanValue } from 'src/utils/entities';
+import { columns } from './constants';
+import { RecentTradesTableProps, TradeMetadata } from './types';
+
+export const RecentTradesTable = ({
+    currency,
+    maturity,
+}: RecentTradesTableProps) => {
+    const [timestamp, setTimestamp] = useState<number>(1643713200);
```

Comment:
what is this hardcoded number?

Reply:
I see it's an initialized value in Advanced Lending etc. before useEffect sets it to "now". Do you think it would be better initialized as 0?

Reply:
Initializing as 0 in my updated changes